### PR TITLE
Add statistics to cli

### PR DESCRIPTION
**Pull request by MaciejWiczk**
Adding: 
- necessary methods in cli, 
- statistics_extractor ,
- statistics_importer, 
- updated tests,
- updated docs

### CODE REVIEW

File: rfhub2/cli/statistics_extractor.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        """
+        execution_time = self.get_execution_time()
+        statistics = [
+            {
+                "collection": k[0],
+                "keyword": k[1],
+                "execution_time": execution_time,
+                "times_used": v,
+                "total_elapsed": 0,
+                "min_elapsed": 2147483647,
```
Comment (pbylicki) [inline rfhub2/cli/statistics_extractor.py]: We should rather use float('inf') as a starting value

File: tests/unit/cli/statistics_extractor.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+STATISTICS_1 = {
+    "collection": "BuiltIn",
+    "keyword": "Log",
+    "execution_time": "2019-12-25 11:38:08.868000",
+    "times_used": 2,
+    "total_elapsed": 2,
+    "min_elapsed": 1,
+    "max_elapsed": 1,
+}
+STATISTICS_2 = {
```
Comment (pbylicki) [inline tests/unit/cli/statistics_extractor.py]: you could reuse first statistics dict fields in the next dicts to make it more concise: STATISTICS_2 = {
        **STATISTICS_1,
        "keyword": "Comment"
    }

File: tests/unit/cli/statistics_importer.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+STATISTICS_1 = {
+    "collection": "BuiltIn",
+    "keyword": "Log",
+    "execution_time": "2019-12-25 11:38:08.868000",
+    "times_used": 2,
+    "total_elapsed": 2,
+    "min_elapsed": 1,
+    "max_elapsed": 1,
+}
+STATISTICS_2 = {
```
Comment (pbylicki) [inline tests/unit/cli/statistics_importer.py]: As above, you could reuse first statistics dict fields in the next dicts to make it more concise

File: rfhub2/cli/api_client.py [hunk 4]
```diff
@@ -56,6 +56,12 @@ def add_keyword(self, data: Dict) -> Dict:
         """
         return self._post_request(endpoint="keywords", data=data)
 
+    def add_statistics(self, data: Dict) -> Response:
```
Comment (pbylicki) [inline rfhub2/cli/api_client.py]: return type is Tuple[int, Dict]

File: rfhub2/cli/statistics_extractor.py (hunk 1 above)
Comment (MaciejWiczk) [reply rfhub2/cli/statistics_extractor.py]: Done

File: tests/unit/cli/statistics_extractor.py (hunk 2 above)
Comment (MaciejWiczk) [reply tests/unit/cli/statistics_extractor.py]: Done

File: tests/unit/cli/statistics_importer.py (hunk 3 above)
Comment (MaciejWiczk) [reply tests/unit/cli/statistics_importer.py]: Done

File: rfhub2/cli/api_client.py (hunk 4 above)
Comment (MaciejWiczk) [reply rfhub2/cli/api_client.py]: Done

File: tests/unit/cli/statistics_extractor.py (hunk 2 above)
Comment (pbylicki) [reply tests/unit/cli/statistics_extractor.py]: it looks like it is still here

File: tests/unit/cli/statistics_importer.py [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+    "keyword": "Log",
+    "execution_time": "2019-12-25 11:38:08.868000",
+    "times_used": 2,
+    "total_elapsed": 2,
+    "min_elapsed": 1,
+    "max_elapsed": 1,
+}
+STATISTICS_2 = {**STATISTICS_1, "keyword": "Comment"}
+STATISTICS_3 = {**STATISTICS_1, "keyword": "Should Be True"}
+STATISTICS_4 = {
```
Comment (pbylicki) [inline tests/unit/cli/statistics_importer.py:25]: this one could also reuse the dict above

File: tests/unit/cli/statistics_extractor.py (hunk 2 above)
Comment (MaciejWiczk) [reply tests/unit/cli/statistics_extractor.py]: this time it's ok :)

**Pull request merged**